### PR TITLE
New product hosting

### PR DESCRIPTION
**Pull request by axxroytovu**

### CODE REVIEW

File: data/ignore.yaml [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+  277019:  "Pure Bulk: Unsorted Commons Unfinity (500ct)"
+  277020:  "Pure Bulk: Unsorted Foil Commons Streets of New Capenna (500ct)"
+  277021:  "Pure Bulk: Unsorted Attractions Unfinity (500ct)"
+  277022:  "Pure Bulk: Unsorted Foil Etched Common and Uncommon Mix Commander Legends: Battle for Baldur's Gate (500ct)"
+  277023:  "Pure Bulk: Unsorted Foil Uncommons Dominaria United (500ct)"
+  277027:  "Pure Bulk: Unsorted Foil Common Unfinity (500ct)"
+  277028:  "Pure Bulk: Unsorted Sticker Cards Unfinity (500ct)"
+  277030:  "Pure Bulk: Unsorted Foil Commons Dominaria United (500ct)"
+  282058:  "Pure Bulk: Wilds of Eldraine Art Cards (500ct)"
+  281245: Wilds of Eldraine Set Booster Variety Pack
```
Comment (kodawah) [inline data/ignore.yaml:199]: should we have a rule against these? this feels like something that will need continuous updating at every set

File: data/products/10E.yaml [hunk 2]
```diff
@@ -0,0 +1,66 @@
+code: 10e
+products:
+  Tenth Edition Booster Box:
+    category: BOOSTER_BOX
+    identifiers:
+      cardKingdomId: '122703'
+      tcgplayerProductId: '27257'
+    purchase_url:
+      cardKingdom: https://www.cardkingdom.com/mtg-sealed/10th-edition-booster-box
```
Comment (kodawah) [inline data/products/10E.yaml:9]: not a fan of having urls in the product spec, they should be derived separately with the identifiers where needed

Comment (axxroytovu) [reply data/products/10E.yaml:9]: Unfortunately Card Kingdom doesn't have programmatically created URLs. These are automatically scraped from their system every day so you don't need to manually add them.

File: data/ignore.yaml (hunk 1 above)
Comment (axxroytovu) [reply data/ignore.yaml:199]: Great future improvement. Right now everything is driven by the ID number, but we could add filtering rules to automatically catch these bulk products.

File: data/products/10E.yaml (hunk 2 above)
Comment (kodawah) [reply data/products/10E.yaml:9]: right but it should be the caller who will rebuild the url by checking the id

File: data/products/GNT.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+code: gnt
+products:
+  Magic Game Night Set:
```
Comment (kodawah) [inline data/products/GNT.yaml:3]: do you think we could keep names unique? it's called in the same way of GN2

Comment (axxroytovu) [reply data/products/GNT.yaml:3]: Moving the products out of this PR, please add this as an issue and I'll make the update.

**Pull request merged**